Start server with quic-agent get error message in console

starting quic-agent, stdout -> /home/webrtc/5.0_package/Release-v5.0ubuntu20210221encrypt/logs/quic-agent.stdout
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module './makeRPC'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/webrtc/5.0_package/Release-v5.0ubuntu20210221encrypt/quic_agent/clusterWorker.js:1:60)
starting management-console, stdout -> /home/webrtc/5.0_package/Release-v5.0ubuntu20210221encrypt/logs/management-console.stdout

--- a/source/agent/quic/dist.json
+++ b/source/agent/quic/dist.json
@@ -18,6 +18,7 @@
           "../../common/clusterWorker.js",
           "../../common/loadCollector.js",
           "../../common/logger.js",
+          "../../common/makeRPC.js",
           "../../../scripts/release/initcert.js",
           "../../../scripts/release/initauth.js",
           "../../../scripts/detectOS.sh"

